### PR TITLE
Fix GPU nightly build

### PR DESCRIPTION
**Pull request by stu1130**
## Channel for questions ##
Before or while filing an issue please feel free to join our [Slack channel](https://join.slack.com/t/deepjavalibrary/shared_invite/zt-ar91gjkz-qbXhr1l~LFGEIEeGBibT7w) to get in touch with development team, ask questions, find out what's cooking and more!

## Description ##
(Brief description of what this PR is about)

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage:
- [ ] Code is well-documented: 
    - For user-facing API changes, Java doc has been updated. 
    - For new examples, README.md is added to explain the what the example does.
- [ ] To the my best knowledge, [examples](https://github.com/awslabs/djl/tree/master/examples) and [jupyter notebooks](https://github.com/awslabs/djl/tree/master/jupyter) are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- [ ] Feature1, tests, (and when applicable, Java doc)
- [ ] Feature2, tests, (and when applicable, Java doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made?
- Interesting edge cases to note here

### CODE REVIEW

**Review by roywei — APPROVED** [review]

File: dlr/dlr-engine/src/test/java/ai/djl/dlr/integraion/DlrTest.java [hunk 1]
```diff
@@ -40,10 +41,11 @@ public void testDlr() throws ModelException, IOException, TranslateException {
         String os;
         if (System.getProperty("os.name").toLowerCase().startsWith("mac")) {
             os = "osx";
-        } else if (System.getProperty("os.name").toLowerCase().startsWith("linux")) {
+        } else if (System.getProperty("os.name").toLowerCase().startsWith("linux")
```
Comment (frankfliu) [inline dlr/dlr-engine/src/test/java/ai/djl/dlr/integraion/DlrTest.java:44]: Why we fail on GPU, if the DLR doesn't GPU, it should fallback to CPU.
It should controlled by getCapadatability() call in the DlrEngine.

Does DLR runtime has to match with the model?
If DLR runtime support GPU, can it run CPU model?
Does that mean if we run one CPU model and one GPU model, we have to load two DLR runtime?

**Review by frankfliu — APPROVED** [review]
